src/ui/public/visualize/loader/errors.ts

```diff
@@ -17,8 +17,22 @@
  * under the License.
  */
 
-import { ElasticsearchError } from './elasticsearch_error';
+import { RequestHandlerParams, Vis } from '../../vis';
 
-export function isTermSizeZeroError(error) {
-  return ElasticsearchError.hasRootCause(error, 'size must be positive, got 0');
+import { toastNotifications } from 'ui/notify';
+
```

Comment:
Unsure if this is the best location for this; thinking it might be better to throw it into a helpers or utils directory, perhaps separating out some of the other pipeline helpers into standalone utils at the same time.